### PR TITLE
Team 3 - Fix claim button

### PR DESCRIPTION
**Pull request by yiivan**
Fixed claim button for non-logged-in users and added test.

### CODE REVIEW

**Comment by yiivan** [comment]
Ready for code review.

File: app/controllers/claim_requests_controller.rb [hunk 1]
```diff
@@ -37,23 +33,24 @@ def update
...
-  
-    redirect_to admin_users_path
-  else
-    flash[:notice] = "No"
-    render admin_users_path
-  end
+      redirect_to admin_users_path
+    else
+      flash[:notice] = "No"
+      render admin_users_path
```
Comment (tkbeili) [inline app/controllers/claim_requests_controller.rb]: Should this be redirect?

File: app/controllers/claim_requests_controller.rb [hunk 2]
```diff
@@ -37,23 +33,24 @@ def update
...
-
   end
 
   private
 
   def find_organization
     @organization = Organization.find params[:organization_id]
   end
+  def find_user
+    @user = User.find params[:user_id]
```
Comment (tkbeili) [inline app/controllers/claim_requests_controller.rb]: Why the need to find the user by `:user_id`? Can you use session?

File: app/views/organizations/show.html.erb [hunk 3]
```diff
@@ -59,13 +56,12 @@
   <%= link_to "Edit", edit_organization_path(@organization), class: "btn btn-primary" %>
 <% end %>
 
-<% if user_signed_in? %>
-  <% if !(@claimed && @claimed.status == true)  %>
-    <% if @organization.user == nil %>
-      <%= link_to "Claim this business", new_organization_claim_request_path(@organization), class: "btn btn-primary"%>
-    <% end %>
+<% if !(@claimed && @claimed.status == true) || @organization.user == nil  %>
```
Comment (tkbeili) [inline app/views/organizations/show.html.erb]: Try to wrap this in a helper method, it's hard to understand what's happening here

File: app/views/organizations/show.html.erb [hunk 4]
```diff
@@ -59,13 +56,11 @@
...
 <% end %>
 
-<% if user_signed_in? %>
-  <% if !(@claimed && @claimed.status == true)  %>
-    <% if @organization.user == nil %>
-      <%= link_to "Claim this business", new_organization_claim_request_path(@organization), class: "btn btn-primary"%>
-    <% end %>
-  <% end %>
+
+<% if !(@claimed && @claimed.status == true) || @organization.user == nil  %>
```
Comment (tkbeili) [inline app/views/organizations/show.html.erb]: No need for `@claimed.status == true` you can simply do `@claimed.status?`. You can also `@organization.user.nil?`

**Comment by tkbeili** [comment]
👍

**Pull request merged**